scheduler/src/main/resources/logback.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE xml>
+<configuration>
+
+	<appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+		<encoder class="ch.qos.logback.classic.encoder.PatternLayoutEncoder">
+			<pattern>%d{yyyy-MM-dd HH:mm:ss} [%thread] %-5level %logger{36} -
+				%msg%n</pattern>
+		</encoder>
+	</appender>
+
+	<!-- -==========Spring========== -->
+	<logger name="org.springframework">
+		<level value="error" />
+	</logger>
+
+	<!-- -==========Quartz========== -->
+	<logger name="org.quartz">
+		<level value="error" />
+	</logger>
+
+	<!-- ===========Hibernate========== -->
+	<!-- Log SQL DML statements as they are executed -->
+	<logger name="org.hibernate.SQL">
+		<level value="debug" />
```

Comment:
I don't think we need more than one. People need to get used to the default and learn how to increase logging only when absolutely necessary. Unless you are going to study the logging, it is useless to generate additional logs that is never looked at.